templates/web/common/page-fragments/head.ftl

```diff
@@ -35,8 +35,12 @@
   <link rel="stylesheet" type="text/css" href="/studio/static-assets/themes/cstudioTheme/css/font-awesome.min.css?version=${UIBuildId!.now?string('Mddyyyy')}" />
   <link rel="stylesheet" type="text/css" href="/studio/static-assets/libs/datetimepicker/jquery.datetimepicker.css?version=${UIBuildId!.now?string('Mddyyyy')}" />
   <link rel="stylesheet" type="text/css" href="/studio/static-assets/styles/uppy.css?version=${UIBuildId!.now?string('Mddyyyy')}" />
+  <link rel="stylesheet" type="text/css" href="/studio/static-assets/styles/video-js.min.css" />
```

Comment:
Try adding this sheet to the scss files by pulling the sheet from the npm package.
Must test that the gradlew build works.

Reply:
not working, we can try it again tomorrow